arduino/main/main.ino

```diff
@@ -25,8 +36,8 @@
 #define debugSerial Serial
 
 // Uncomment whatever type you're using!
-#define DHTTYPE DHT11   // DHT 11
-//#define DHTTYPE DHT22   // DHT 22  (AM2302), AM2321
+//#define DHTTYPE DHT11   // DHT 11
```

Comment:
If this won't be used, might as well delete it

Reply:
Right now we have one device that is using the DHT11 and the other is using the DHT22 - I think we should keep that comment there for now

Reply:
Sounds good!